pkg/reconcilermanager/controllers/rootsync_controller.go

```diff
@@ -1029,22 +1030,22 @@ func (r *RootSyncReconciler) updateSyncStatus(ctx context.Context, rs *v1beta1.R
 	return updated, nil
 }
 
-// enableRendering returns whether rendering should be enabled for the reconciler
-// of this RSync. This is determined by an annotation that is set on the RSync
-// by the reconciler.
-func enableRendering(annotations map[string]string) bool {
-	val, ok := annotations[metadata.RequiresRenderingAnnotationKey]
-	if !ok { // default to disabling rendering
+// enableAnnotation returns whether the annotation should be enabled for the
+// reconciler of this RSync. This is determined by an annotation that is set on
+// the RSync by the reconciler.
+func enableAnnotation(key string, annotations map[string]string) bool {
```

Comment:
This reads like a "set" instead of a "get" function. Suggest `checkAnnotationEnabled`.

Reply:
Or make it shorter, `isAnnotationEnabled` or just `annotationEnabled `

Reply:
done